nexus/types/src/external_api/headers.rs

```diff
@@ -0,0 +1,20 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+use schemars::JsonSchema;
+use serde::Deserialize;
+use serde::Serialize;
+
+/// Range request headers
+#[derive(Debug, Deserialize, Serialize, JsonSchema)]
+pub struct RangeRequest {
+    /// A request to access a portion of the resource, such as:
+    ///
+    /// ```
```

Comment:
Failing doc test

```suggestion
    /// ```text
```

Reply:
Patched!